CHANGELOG.md

```diff
@@ -5,9 +5,11 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased]
-### Uncategorized
-- Add support for send call with callback so that Web3 Providers can be used ([#97](https://github.com/MetaMask/eth-json-rpc-middleware/pull/97))
-- Add support for `eth_signTransaction` so that external key stores can be used for signing ([#96](https://github.com/MetaMask/eth-json-rpc-middleware/pull/96))
+
+## [7.1.0]
+### Added
+- Add `send` method to provider and `ethersProviderAsMiddleware` ([#97](https://github.com/MetaMask/eth-json-rpc-middleware/pull/97))
+- Add `eth_signTransaction` support ([#96](https://github.com/MetaMask/eth-json-rpc-middleware/pull/96))
```

Comment:
We might want to consider this as breaking, since it would in effect break both of our products if we didn't explicitly block this method first.